packaging/Unix.cmake

```diff
@@ -4,8 +4,8 @@ set(CPACK_COMPONENTS_HALIDE_RUNTIME Halide_Runtime)
 set(CPACK_COMPONENTS_HALIDE_DEVELOPMENT Halide_Development)
 
 set(CPACK_INSTALL_CMAKE_PROJECTS
-    static-Debug Halide ALL /
-    shared-Debug Halide ALL /
+#    static-Debug Halide ALL /
```

Comment:
I presume this is intentional, but a comment needs to be added saying why they are disabled

Reply:
Done